trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/view/ApplicationStartUpDataFormatterTest.java

```diff
@@ -0,0 +1,77 @@
+package io.bitrise.trace.data.management.formatter.view;
+
+import com.google.protobuf.Timestamp;
+
+import org.junit.Test;
+
+import io.bitrise.trace.data.collector.view.ApplicationStartUpDataListener;
+import io.bitrise.trace.data.dto.ApplicationStartData;
+import io.bitrise.trace.data.dto.ApplicationStartType;
+import io.bitrise.trace.data.dto.Data;
+import io.bitrise.trace.data.dto.FormattedData;
+import io.bitrise.trace.data.management.formatter.BaseDataFormatterTest;
+import io.opencensus.proto.metrics.v1.LabelKey;
+import io.opencensus.proto.metrics.v1.LabelValue;
+import io.opencensus.proto.metrics.v1.Metric;
+import io.opencensus.proto.metrics.v1.MetricDescriptor;
+import io.opencensus.proto.metrics.v1.Point;
+import io.opencensus.proto.metrics.v1.TimeSeries;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for {@link ApplicationStartUpDataFormatter}
+ */
+public class ApplicationStartUpDataFormatterTest extends BaseDataFormatterTest {
+
+    final ApplicationStartUpDataFormatter formatter = new ApplicationStartUpDataFormatter();
+    final ApplicationStartData applicationStartData = new ApplicationStartData(1234, ApplicationStartType.COLD);
+
+    @Test
+    public void formatData_notApplicationStartUp() {
+        final Data data = new Data(ApplicationStartUpDataListener.class);
+        data.setContent("kittens");
+        assertArrayEquals(new FormattedData[]{}, formatter.formatData(data));
+    }
+
+    @Test
+    public void formatData() {
```

Comment:
Test method is pretty long, might worth to create a different method for creating the sample/expected data. What do you think?